### DISCUSSION TITLE
Customize some terms such as "toot" and "boost" for the sake of branding

### DISCUSSION THREAD

**Discussion by robgitnyc** [mastodon/mastodon#21538] [General]
I would like to custmize terms such as "boost" and "toot" for something that makes sense with the whole story/brand of my instance. Where can I modify that? I looked up at config/locales but it wasn't there.

**Comment by robgitnyc** [comment]
Also, tried editing ~/live/app/javascript/mastodon/locales/es.json (the default language of my instance), but changes made to the file are not reflected in the app after restarting it.

**Reply by NicolaeNMV** [reply]
@robgitnyc just restarting the app may be not enough, as those files are "compiled" into one file. I think it's this step that does "Run asset precompilation" that you need to run https://docs.joinmastodon.org/admin/install/#generating-a-configuration Good luck !
